tests/CacheCompat/CommonCache.Test.Common/LabUserData.cs

```diff
@@ -1,21 +1,25 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+using System;
+
 namespace CommonCache.Test.Common
 {
     public class LabUserData
```

Comment:
is it not possible to use the lab response classes that already have this data? This class seems redundant unless I am missing something?

Reply:
I originally thought the same, but this class is more of an adapter for the Lab objects. This gets serialized and re-serlialized to JSON and passed to python and to java. Also, I'd need to make changes to each cache test project to take a dependnecy on LabInfra dll, which is not trivial. 

So all in all - yes, I agree with you that this is a redundant object, but in practical terms it simplifies the design of the cache tests.